config/locales/en/summary.yml

```diff
@@ -380,3 +406,45 @@ en:
         answers: *OWNERSHIP_TYPE 
         absence_answer: *absence_none
       # END investments
+
+      # BEGIN property, address and owners
+      house_type:
+        question: Which type of property is it?
+        absence_answer: *absence_none
+      bedrooms:
+        question: How many bedrooms are there?
+        absence_answer: *absence_none
+      value:
+        question: How much is the property worth?
+        absence_answer: *absence_none
+      outstanding_mortgage:
+        question: How much is left to pay on the mortgage?
+        absence_answer: *absence_none
+      percentage_applicant_owned:
+        question: What percentage of the property does your client own?
+        absence_answer: *absence_none
+      percentage_partner_owned:
+        question: What percentage of the property does your client’s partner own?
+        absence_answer: *absence_none
+      is_home_address:
+        question: Is the address of the property the same as your client’s home address?
+        answers: *YESNO
+        absence_answer: *absence_none
+      has_other_owners:
+        question: Does anyone else own part of the property?
+        answers: *YESNO
+        absence_answer: *absence_none
+      address:
+        question: Address
+        absence_answer: *absence_none
+      name:
+        question: What is the name of the [%{index}] other owner?
```

Comment:
Remove square brackets - these were just to show in the design that we wanted an index

Reply:
That needs amending in the helpers yaml too if it's not planned for another PR

```
steps_capital_property_owner_form:
        property_owners_attributes:
          name: What is the name of the [first] other owner?
```

You should be able to use Mo's integer.yml to convert them